packages/slice-machine/lib/builders/CustomTypeBuilder/Layout/Header.tsx

```diff
@@ -2,18 +2,18 @@ import React from "react";
 import { Box, Button, Spinner, Text } from "theme-ui";
 
 import Header from "../../../../components/Header";
-import useSliceMachineActions from "@src/modules/useSliceMachineActions";
```

Comment:
It's not the first time I see a change of import in your PR, is there a specific reason to do that ?

I mean it's not like the code didn't work before so is it because you prefer relative path ?

Reply:
Multiple reasons.
+ some times the alias path is longer.
+ my ide doesn't work with them.
+ the paths aren't resolved at compile time, so global module resolution is modified by the `module-alias` lib
+ `module-alias` doesn't work in projects that use hoisting. https://github.com/ilearnio/module-alias/issues/118
+ Node can do this natively https://github.com/ilearnio/module-alias/issues/113

Reply:
- sometimes the alias path is long ✅ 
- my ide doesn't work with them 🟥 , pretty sure your IDE should be able to do that
- use of module alias, I do agree that is not a good practice.

Overall, if the path is long I would say ok but here replacing "@src" with "../../../../" doesn't seems right and honestly the aliases looses their purpose.

Reply:
> my ide doesn't work with them 🟥 , pretty sure your IDE should be able to do that
Well it doesn't.